### PR TITLE
SPLICE-2251 platform_it/pom.xml cleanup

### PR DESCRIPTION
**Pull request by OlegMazurov**
Don't copy 'test' scope artifacts to target/dependency (used by Yarn, Spark).
Minimize dependencies.

### CODE REVIEW

**Comment by OlegMazurov** [comment]
jenkins please test branch @cdh5.14.0

**Comment by spliceci** [comment]
TEST FAILED

File: platform_it/pom.xml [hunk 1]
```diff
@@ -491,169 +122,85 @@
             </exclusions>
         </dependency>
         <dependency>
-            <groupId>org.apache.hadoop</groupId>
-            <artifactId>hadoop-common</artifactId>
-            <version>${hadoop.version}</version>
```
Comment (bklo94) [inline platform_it/pom.xml]: I believe this needs to be included or at least the hadoop-aws dependency needs to be added in. I don't believe we have working IT's to test S3. I tested it by running "standalone" (start-splice-server) and ran it on 5.8.3 and 5.14.0 on an anonymous import.
```
splice> CREATE TABLE CUSTOMERS(CUSTOMER_MASTER_ID BIGINT PRIMARY KEY);
0 rows inserted/updated/deleted
splice> call SYSCS_UTIL.IMPORT_DATA('SPLICE' , 'CUSTOMERS', null , 's3a://splice-demo/demodata/data/customers.csv', ',' , null , null , 'yyyy-MM-dd' , null , 0 , null , null , null);
ERROR EXT34: Error when accessing S3:splice.aws.com.amazonaws.SdkClientException: Unable to load credentials from service endpoint
```


I compared it to master's and master's, which is currently working

```
splice> CREATE TABLE CUSTOMERS(CUSTOMER_MASTER_ID BIGINT PRIMARY KEY);
0 rows inserted/updated/deleted
splice> call SYSCS_UTIL.IMPORT_DATA('SPLICE' , 'CUSTOMERS', null , 's3a://splice-demo/demodata/data/customers.csv', ',' , null , null , 'yyyy-MM-dd' , null , 0 , null , null , null);
rowsImported        |failedRows          |files      |dataSize            |failedLog
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
1000000             |0                   |1          |7920000             |NONE

1 row selected
splice> SELECT COUNT(*) FROM CUSTOMERS;
1
--------------------
1000000

1 row selected
```

Comment (OlegMazurov) [reply platform_it/pom.xml]: It works for me (cdh5.13.3). What do you mean by "anonymous import"? Do you disable ~/.aws/credentials?

Comment (bklo94) [reply platform_it/pom.xml]: I a clean and force update on the build and it seemed to have fixed my error. Odd that it didn't update. What anonymous import means is after DBAAS-1817 if you don't supply credentials for an AWS import, presto falls back and uses AWS's anonymous credentials api to get credentials for public bucket access. 

+1

**Comment by OlegMazurov** [comment]
jenkins please test branch @cdh5.13.3

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

**Pull request merged**